applications/IOIOManager/src/main/java/ioio/manager/ProgrammerActivity.java

```diff
@@ -75,7 +75,7 @@ public static Chip findById(int id) {
 			return UNKNOWN;
 		}
 
-		private Chip(int id) {
+		Chip(int id) {
```

Comment:
These too. I'm curious why it was necessary to remove a level of encapsulation here.

Reply:
It was pointless, because it's package protected